### ISSUE TITLE
Uncaught ReferenceError: fs is not defined

### ISSUE THREAD

**Issue by xxdondi** [hokaccha/linter-coffeescript#1]
Whenever I open a `.coffee` file in atom, I get this error. Any ideas?

**Atom Version**: 0.179.0
**System**: linux 3.13.0-52-generic
**Thrown From**: [linter-coffeescript](https://github.com/hokaccha/linter-coffeescript) package, v0.1.1
### Stack Trace

Uncaught ReferenceError: fs is not defined

```
At /home/cracker/.atom/packages/linter-coffeescript/lib/linter-coffeescript.coffee:11

ReferenceError: fs is not defined
  at LinterCoffeeScript.lintFile (/home/cracker/.atom/packages/linter-coffeescript/lib/linter-coffeescript.coffee:11:5)
  at /home/cracker/.atom/packages/linter/lib/linter-view.coffee:163:18
  at Array.forEach (native)
  at /home/cracker/.atom/packages/linter/lib/linter-view.coffee:162:18
  at FSReqWrap.oncomplete (fs.js:100:15)

```
### Commands

```
     -0:12.2 application:open-folder (atom-text-editor.editor)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
autocomplete-plus, v2.2.0
linter, v0.12.2
linter-coffeescript, v0.1.1
minimap, v4.3.1

# Dev
No dev packages
```

**Comment by VasylMarchuk** [comment]
I'm also having the same issue:

```
/Volumes/Home/VASMAN/.atom/packages/linter-coffeescript/lib/linter-coffeescript.coffee:11
Hide Stack Trace
ReferenceError: fs is not defined
  at LinterCoffeeScript.lintFile (/Volumes/Home/VASMAN/.atom/packages/linter-coffeescript/lib/linter-coffeescript.coffee:11:5)
  at /Volumes/Home/VASMAN/.atom/packages/linter/lib/linter-view.coffee:163:18
  at Array.forEach (native)
  at /Volumes/Home/VASMAN/.atom/packages/linter/lib/linter-view.coffee:162:18
  at FSReqWrap.oncomplete (fs.js:77:15)
```

##### `apm ls`

```
/Users/VASMAN/.atom/packages (15)
├── file-icons@1.5.5
├── isotope-ui@2.2.1
├── language-coffee-script@0.39.0
├── language-emblem@0.1.0
├── linter@0.12.2
├── linter-coffeescript@0.1.1
├── linter-jshint@0.1.4
├── merge-conflicts@1.3.1
├── minimap@4.8.0
├── monokai@0.14.0
├── sepia-syntax@1.0.5
├── seti-monokai@0.3.1
├── seti-syntax@0.4.0
├── seti-ui@0.7.1
└── symbols-tree-view@0.9.2
```

##### `apm --version`

```
apm  0.167.0
npm  2.5.1
node 0.10.35
python 2.7.6
git 2.3.2
```

##### `atom --version`

```
0.199.0
```

**Comment by hokaccha** [comment]
This issue is fixed at v0.1.2. Thanks!

**Comment by arjes** [comment]
Looks like this exists again in 0.2.0

**Atom Version**: 0.201.0
**System**: Mac OS X 10.10.3
**Thrown From**: [linter-coffeescript](https://github.com/hokaccha/linter-coffeescript) package, v0.2.0

### Stack Trace

Uncaught ReferenceError: fs is not defined

```
At ~/.atom/packages/linter-coffeescript/lib/linter-coffeescript.coffee:24

ReferenceError: fs is not defined
  at LinterCoffeeScript.lintFile (~/.atom/packages/linter-coffeescript/lib/linter-coffeescript.coffee:7:3)
  at /~/.atom/packages/linter/lib/linter-view.coffee:163:18
  at Array.forEach (native)
  at ~/.atom/packages/linter/lib/linter-view.coffee:162:18
  at FSReqWrap.oncomplete (fs.js:77:15)

```

### Commands

```
     -6:51.3.0 editor:select-to-first-character-of-line (atom-text-editor.editor.mini.is-focused)
  5x -6:51.1.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -6:46.6.0 editor:move-to-first-character-of-line (atom-text-editor.editor.mini.is-focused)
  2x -6:44.7.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -6:43.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -4:17.3.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -4:14.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -4:13.9.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -4:13.9.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -2:01.5.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
  3x -1:59 core:backspace (atom-text-editor.editor.mini.is-focused)
     -1:58.4.0 editor:select-to-first-character-of-line (atom-text-editor.editor.mini.is-focused)
     -1:58.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -1:54.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:53.8.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -1:51.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

``` json
{
  "core": {}
}
```

### Installed Packages

``` coffee
# User
atom-beautify, v0.27.6
linter, v0.12.6
linter-coffeescript, v0.2.0
linter-ruby, v0.1.6

# Dev
No dev packages
```
